examples/debug_info.py

```diff
@@ -25,7 +25,7 @@
 folder = os.path.dirname(os.path.abspath(__file__))
 sys.path.insert(0, os.path.normpath(f"{folder}/.."))
 
-# pylint: disable=import-error, wrong-import-position
+# pylint: disable=import-error, wrong-import-position, useless-suppression
```

Comment:
The PR description doesn't explain why we need to keep the disable of import error.

Reply:
If I keep only this:
`# pylint: disable=wrong-import-position`

And commit, I get this:

```
************* Module debug_info
examples/debug_info.py:30:0: E0401: Unable to import 'pytradfri' (import-error)
examples/debug_info.py:31:0: E0401: Unable to import 'pytradfri.api.libcoap_api' (import-error)
examples/debug_info.py:32:0: E0401: Unable to import 'pytradfri.error' (import-error)
examples/debug_info.py:33:0: E0401: Unable to import 'pytradfri.util' (import-error)
```

So I revert back to this line:
`# pylint: disable=import-error, wrong-import-position`

And get this:
```
(.venv) vscode ➜ /workspaces/pytradfri (add-pylint-ignores) $ pylint pytradfri examples tests
************* Module debug_info
examples/debug_info.py:28:0: I0021: Useless suppression of 'import-error' (useless-suppression)
************* Module example_air_purifier_async
examples/example_air_purifier_async.py:26:0: I0021: Useless suppression of 'import-error' (useless-suppression)
************* Module example_async
examples/example_async.py:26:0: I0021: Useless suppression of 'import-error' (useless-suppression)
************* Module example_cover_async
examples/example_cover_async.py:26:0: I0021: Useless suppression of 'import-error' (useless-suppression)
************* Module example_pair
examples/example_pair.py:31:0: I0021: Useless suppression of 'import-error' (useless-suppression)
************* Module example_socket_async
examples/example_socket_async.py:26:0: I0021: Useless suppression of 'import-error' (useless-suppression)
************* Module example_sync
examples/example_sync.py:27:0: I0021: Useless suppression of 'import-error' (useless-suppression)
```

As per this per this PR, I now make the ignore line like so: `# pylint: disable=import-error, wrong-import-position, useless-suppression`

```
(.venv) vscode ➜ /workspaces/pytradfri (add-pylint-ignores) $ pylint pytradfri examples tests
(.venv) vscode ➜ /workspaces/pytradfri (add-pylint-ignores) $ pre-commit run --hook-stage manual pylint --all-files --show-diff-on-failure
pylint...................................................................Passed
```

Reply:
I'm a bit puzzled that this error appears all of a sudden though.

Reply:
I think the main problem is that we try to run the script without installing the pytradfri package properly and instead do the custom injection of the library on the path. It would be simpler to ask users to install the package and set up an entry point for all scripts when installing the package.

I think it's ok to merge this PR for now though.